docs/configuration.md

```diff
@@ -13,19 +13,13 @@ metadata:
   name: k0s
 spec:
   api:
-    externalAddress: my-lb-address.example.com
```

Comment:
I'm assuming this was just an example so you had to replace it

Reply:
Yeah, I just put the default config. I add the example back.